### PR TITLE
Dunitz/cellxgene db  model import

### PR DESCRIPTION
**Pull request by MDunitz**
## Need
- To add cellxgene tables to corpora DB

## Approach
- Import from cellxgene package and create additional base

## Testing
- Drop/Create tables
- Add rows to cellxgene tables
- Check local db to ensure rows created correctly

## Note
- Do not merge until updated cellxgene package is released

### CODE REVIEW

File: backend/scripts/create_db.py [hunk 1]
```diff
@@ -10,16 +10,29 @@
...
+from .corpora_config import CorporaDbConfig
+from .corpora_orm import Base
+
+
+from sqlalchemy.schema import DropTable
+from sqlalchemy.ext.compiler import compiles
+from server.db.cellxgene_orm import Base as CellxGeneBase
+
+
+@compiles(DropTable, "postgresql")
```
Comment (Bento007) [inline backend/scripts/create_db.py:22]: So what all is happening in this function?

File: backend/scripts/create_db.py [hunk 2]
```diff
@@ -10,16 +10,29 @@
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from corpora.common.corpora_config import CorporaDbConfig
-from corpora.common.corpora_orm import Base
+from .corpora_config import CorporaDbConfig
+from .corpora_orm import Base
```
Comment (Bento007) [inline backend/scripts/create_db.py:14]: I recommend renaming to `CorporaBase` for clarity

File: backend/database/versions/f6b189a1d9c1_create_cellxgene_tables.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+from sqlalchemy import Column, String, TIMESTAMP, func
+
+revision = 'f6b189a1d9c1'
+down_revision = '899656d37baa'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
```
Comment (Bento007) [inline backend/database/versions/f6b189a1d9c1_create_cellxgene_tables.py:22]: wouldn't a cellxgene user be the same thing as a corpora user according to the DB?

File: backend/scripts/create_db.py (hunk 1 above)
Comment (MDunitz) [reply backend/scripts/create_db.py:22]: It appends cascade to the drop table command to handle foreign keys/dependencies

**Pull request closed without merging**